bin/rails

```diff
@@ -1,6 +1,4 @@
 #!/usr/bin/env ruby
-# frozen_string_literal: true
-
-APP_PATH = File.expand_path('../config/application', __dir__)
-require_relative '../config/boot'
-require 'rails/commands'
+APP_PATH = File.expand_path("../config/application", __dir__)
+require_relative "../config/boot"
+require "rails/commands"
```

Comment:
Do you understand why Rubocop has changed this now? In main this file is using single quotes.

Reply:
Or maybe Rubocop hasn't touched it, and this is what's been brought in by `rails app:update`. I think it'd be good to switch from double to single quotes in this commit so that the diff is easier to parse. What do you think?

Reply:
I'm afraid I don't. In general I'm happy to do whatever makes rubocop happy, so I won't look into it much further unless you're worried it's hiding something?

Reply:
I had a play. We're seeing this change from single to double quotes because that's how Rails generates these files, and running `rubocop -A` doesn't fix them automatically. You can fix them by running rubocop for the specific files, e.g. `rubocop -A bin/rails`. I think it's worth doing to avoid introducing unnecessary noise in this commit.

Reply:
Thanks for investigating @chrisroos! I've fixed those in ea3d2f9